modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/view/table/ClayTableSchema.java

```diff
@@ -21,11 +21,11 @@
  */
 public class ClayTableSchema {
 
-	public Map<String, ClayTableSchemaField> getFields() {
+	public Map<String, ClayTableSchemaField> getClayTableSchemaFields() {
```

Comment:
hey @brianchandotcom we are returning a Map here and not a ClayTableSchemaField. Do we still need to rename this?

Reply:
Oh, looking more on the codebase I see the pattern.. Will resend